### ISSUE TITLE
Full accessibility testing

### ISSUE THREAD

**Issue by torrin47** [USEPA/EPA_Non-geo_Metadata_Editor#102]
Potentially more we could do to assist with screenreaders and 508 compliance? Testing?

**Comment by torrin47** [comment]
Limited testing/scope of work eval using chrome extensions.

**Comment by torrin47** [comment]
Tried scanning site with Siteimprove Accessibility Checker Chrome Extension.  Keyboard navigation was strong, only area of concern was lack of aria-label for input fields.  We use defaultText as the effective labeling for these fields, I wonder whether there's a way to avoid duplication and place the same value in both attributes?

**Comment by aergul** [comment]
We made a number of changes based on several tools' recommendations. Siteimprove was one of them but must be conflicting with something as it only ran a handful of times - froze up mostly. We can revisit later if needed.

**Comment by torrin47** [comment]
Awesome, thanks!  I think that should suffice and we can consider this closed.
